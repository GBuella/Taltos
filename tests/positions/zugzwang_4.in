force
nopost
setboard 8/6B1/p5p1/Pp4kp/1P5r/5P1Q/4q1PK/8 w - -
nps 100000
st 200
search_sync
hash_value_min 300
