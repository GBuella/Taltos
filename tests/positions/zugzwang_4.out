1. Qxh4+
ok
